source/nodejs/adaptivecards/src/card-elements.ts

```diff
@@ -6660,6 +6660,11 @@ export abstract class ContainerWithActions extends Container {
 
         if (element) {
             let renderedActions = this._actionCollection.render(this.hostConfig.actions.actionsOrientation, false);
+            // we can't set rtl at the card level if the card has carousel, setting the rtl explicitly at the action container takes 
+            // care of card level rtl for actions
+            if (this.rtl !== undefined && renderedActions) {
```

Comment:
Can we move this to AdativeCards, likely AdaptiveCards has knowledge about carousel and rtl

Reply:
Why would this be necessary? The `renderedActions` element already inherits the `rtl` property of its parent container.

Reply:
this is addressed in #6723